pkg/southbound/gnmi/conn_manager.go

```diff
@@ -38,28 +36,27 @@ import (
 type ConnManager interface {
 	Get(ctx context.Context, targetID topoapi.ID) (Conn, error)
 	List(ctx context.Context) ([]Conn, error)
-	Watch(ctx context.Context, ch chan<- Conn) error
+	Watch(ctx context.Context, ch chan<- ConnEvent) error
 	Connect(ctx context.Context, target *topoapi.Object) (Conn, error)
+	Remove(ctx context.Context, targetID topoapi.ID) error
```

Comment:
I think if we’re going to call one method `Connect`, this method should be called `Close`. Alternatively, we could exclude this method and just remove it from the manager when the `Conn`’s `Close` method is called. Either way, the package should only export one method for closing/removing a connection — either Conn’s `Close` method or ConnManager’s `Close` method.

Reply:
Renamed to Close